pkg/perf/cpubench_task.go

```diff
@@ -0,0 +1,75 @@
+package perf
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+
+	"github.com/threefoldtech/zos/pkg/stubs"
+)
+
+// CPUBenchmarkTask defines CPU benchmark task data.
+type CPUBenchmarkTask struct {
+	taskID   string
+	schedule string
+}
+
+// CPUBenchmarkResult holds CPU benchmark results with the workloads number during the benchmark.
+type CPUBenchmarkResult struct {
```

Comment:
Is the Single/Multi process or a thread? can we add a few more info and even change the name to a include the full name

Reply:
I can change the name of the fields but the data returned from `zos.perf.get_all` will still be single, multi and threads, is that fine?